src/sample/stuDashController.java

```diff
@@ -104,11 +124,33 @@ private void ChoiceBoxItem() {
     programme.getItems().addAll("BSc Computer Science", "BSc Mathematics", "BSc Management");
     major.getItems().addAll("Software Engineering", "Intelligent Computing", "Computing Infrastructure");
     minor.getItems().addAll("Accounting", "Mathematics", "Electives");
+    
+
+  }
+
+  public void display() throws IOException{
+    File stuinfoFile = new File("data/Student Dashboard/"+name.getText()+".txt");
```

Comment:
You shouldn't use name as the name of txt file

Reply:
testing purposes, thats why i use name as name of txt file